data/selinux/snappy.te

```diff
@@ -504,6 +503,9 @@ userdom_delete_user_tmp_files(snappy_mount_t)
 files_manage_generic_tmp_dirs(snappy_mount_t)
 files_manage_generic_tmp_files(snappy_mount_t)
 
+# Allow snap-{update,discard}-ns to mount on /etc/
```

Comment:
Where are you getting the `snap-{update,discard}-ns` come from?

Reply:
As discussed, snap-confine uses snap-update-ns to actually do the mounts